vividus-plugin-mobile-app/src/main/java/org/vividus/mobileapp/steps/TouchSteps.java

```diff
@@ -121,25 +121,23 @@ public void doubleTapByLocator(Locator locator)
     public void swipeToElement(SwipeDirection direction, Locator locator, Duration swipeDuration)
     {
         locator.getSearchParameters().setWaitForElement(false);
-        Supplier<Rectangle> swipeArea = createSwipeArea();
+        Rectangle swipeAreaRectangle = createSwipeArea().get();
```

Comment:
update `createSwipeArea` to return actual rectangle instead of Supplier

Reply:
done